src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from 'react';
+import {action, computed, observable} from 'mobx';
+import {observer} from 'mobx-react';
+import selectStyles from './select.scss';
+import Divider from './Divider';
+import OverlayList from './OverlayList';
+import Option from './Option';
+import Label from './Label';
+
+@observer
+export default class Select extends React.PureComponent {
+    props: {
+        value?: string,
+        onChange?: (s: string) => void,
```

Comment:
Now I remember. It is not flow that causes the problem, but styleguidist or react-docgen to be more specific. If you don't give a name to the types react docgen outputs the following error and the component isn't displayed in the styleguide.

```
Warning: Cannot parse src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js: TypeError: Cannot read property 'indent' of undefined
```